developer/src/kmc-keyboard/test/tsconfig.json

```diff
@@ -20,6 +20,7 @@
   "references": [
       { "path": "../../../../common/web/keyman-version/tsconfig.esm.json" },
       { "path": "../../../../common/web/types/" },
+      { "path": "../../../../common/tools/hextobin/" },
```

Comment:
This is needed to fix a build failure that arises due to order of compilation, otherwise.